doc/source/parameters/resolved_cfd-dem/resolved_cfd-dem.rst

```diff
@@ -157,15 +163,38 @@ The following parameter and subsection are all inside the subsection ``particle
+* If the parameter ``integrate motion`` is set to ``true``, the subsection ``omega`` defines the initial value of the particle's rotational velocity. Otherwise, it defines the particle's rotational velocity at all time steps. This rotational velocity is expressed as a function that can evolve in time. Each component of the ``Function expression`` corresponds to the value of its component in the X, Y, and Z directions. It's important to note that even the 2D solver uses the rotational velocity in 3D. In this case, it will only use the Z component of the rotational velocity.
+
+* The subsection ``orientation`` defines the initial value of the particle's angular position around each of the axes X, then Y, and lastly Z.
+
+.. warning::
+    It's important to note that even the 2D solver uses the rotational velocity in 3D. In that case, it will only use the Z component of the rotational velocity, but all three should be defined.
 
 * The ``inertia`` parameter is used to define one of the diagonal elements of the rotational inertia matrix. Since we are defining spherical particles, we assume a uniform distribution of mass, and as such, all the diagonal elements of the rotational inertia matrix are the same.
 
 * The ``pressure x``, ``pressure y``, and ``pressure z`` parameters are used to define the X, Y, and Z coordinate offset of the pressure reference point relative to the center of the particle. These parameters are used when the ``assemble Navier-Stokes inside particles`` parameter is set to true to define the pressure reference point.
 
-* The ``radius`` parameter is used to define the radius of this particle.
+* The ``type`` parameter is used to define the geometry type of the particle. The alternatives in 2D are: ``sphere``, ``ellipsoid``, ``rectangle``. In 3D, in addition to the previous shapes, alternatives include: ``cone``, ``death star``, ``cut hollow sphere``, ``torus``.
+
+* The subsection ``solid arguments`` is used to define the parameters of the solid in the form of a tri-components function. The arguments, which must be written in triplets, are:
+    * Sphere: radius, [none], [none];
+
+    * Rectangle: x half length, y half length, z half length;
+
+    * Ellipsoid: x radius, y radius, z radius;
+
+    * Torus: torus radius, torus thickness radius, [none];
+
+    * Cone: tan(base angle), height, [none];
+
+    * Cut Hollow Sphere: radius, cut height, wall thickness;
+
+    * Death Star: sphere radius, hole radius, distance between centers.
```

Comment:
Why use triplets? It would be a lot better to use a list of strings and, depending on the shape and the number of arguments required, parse the arguments. This way you could have a variable number of arguments depending on the shape. You can look at the DEM list insertion mechanism to see how this can be done (or come talk to me, I'll show you the code :) )